src/components/dialogs/GenericErrorDialog.vue

```diff
@@ -4,7 +4,6 @@
     width="45rem"
     persistent
     :attach="attach"
-    content-class="delete-error-dialog"
```

Comment:
Please add this back in as "generic-error-dialog" -- it can be used for unit testing.